app/lang/fr.json

```
@@ -48,7 +48,7 @@
   "An error has occurred": "Une erreur est survenue",
   "An unknown error has occurred": "Une erreur inconnue s'est produite",
   "An update to Mailspring is available %@": "Une mise à jour de Mailspring est disponible %@",
-  "Any": "Tout",
+  "Any": "N'importe lequel",
```

Since the french translation is `Si %@ des conditions suivantes sont remplies :`, `any` should be feminine because conditions is feminine.
```suggestion
  "Any": "N'importe laquelle",
```